.gitmodules

```diff
@@ -1,7 +1,8 @@
 [submodule "src/libmongoc"]
 	path = src/libmongoc
-	url = https://github.com/mongodb/mongo-c-driver.git
+	url = https://github.com/jmikola/mongo-c-driver.git
 	ignore = untracked
+	branch = 2.1.2-handshake
```

Comment:
Is the fix you added on [your branch](https://github.com/jmikola/mongo-c-driver/commits/2.1.2-handshake/) necessary for this PR?